.werft/build.ts

```diff
@@ -406,8 +413,10 @@ export async function deployToDev(deploymentConfig: DeploymentConfig, workspaceF
             flags += ` -f ../.werft/values.dev.gcp-storage.yaml`;
         }
 
+        let nodepoolNumber = parseInt(createHash('sha256').update(namespace).digest('hex').substring(0,5),16) % nodeAffinityValues.length;
```

Comment:
nit: `nodepoolIndex` would be more precise/expected.
nit: `const`

Also, we definitely need at least a short sentence to explain what we're doing.

Reply:
Done